Lib/test/test_minidom.py

```diff
@@ -1560,5 +1562,28 @@ def testProcessingInstructionNameError(self):
         pi = doc.createProcessingInstruction("y", "z")
         pi.nodeValue = "crash"
 
+    def test_minidom_attribute_order(self):
+        xml_str = '<?xml version="1.0" ?><curriculum status="public" company="example"/>'
+        doc = parseString(xml_str)
+        output = io.StringIO()
+
+        with contextlib.redirect_stdout(output) as xml_file:
+            doc.writexml(xml_file)
+        self.assertEqual(output.getvalue(),
+                         '<?xml version="1.0" ?><curriculum status="public" company="example"/>')
```

Comment:
Is it `xml_str`? Maybe use this variable?

Reply:
You are right, is redundant. I thought to do it with a different object to be sure of the output.